starlite/app.py

```diff
@@ -12,6 +12,8 @@
 from starlette.types import ASGIApp, Receive, Scope, Send
 from typing_extensions import Type
 
+from rust_backend import RouteMap as RouteMapInit
```

Comment:
Maybe,  change the module name from `rust_backend` to something like `starlite._rust` or `starlite._route_map`?

Reply:
I guess we don’t have to if there are stubs